prog/pwm/fancontrol.8

```diff
@@ -109,12 +114,17 @@ loaded kernel drivers.
 .SH THE ALGORITHM
 
 \fBfancontrol\fP first reads its configuration, writes it to arrays and loops its
-main function.  This function gets the temperatures and fanspeeds from
+main function. This function gets the temperatures and fanspeeds from
 kernel driver files and calculates new speeds depending on temperature
 changes, but only if the temp is between MINTEMP and MAXTEMP. After that, the
-new values are written to the PWM outputs. Currently the speed increases
-quadratically with rising temperature. This way you won't hear your fans most
-of the time at best.
+new values are written to the PWM outputs. The fan speed increases
+linearly with rising temperature. Optional configuration enables quadratic
+increase. 
+.IP
+FCCHARCUR=hwmon0/device/pwm1=SQR
+.PP
+Quadratic curve may bring the sound level further down, with accepting higher temperature.
+It may  make sense in systems with predominatly very low thermal load.
```

Comment:
"... may reduce sound level further while accepting higher system temperature".
s/It/This/
double space between may and make
s/predominatly/predominantly/
s/very low/low/